src/OWML.ModHelper.Events/Patches.cs

```diff
@@ -39,7 +39,7 @@ internal static class Patches
 		public static void AfterDestroy(MonoBehaviour __instance) =>
 			OnEvent?.Invoke(__instance, Common.Events.AfterDestroy);
 
-		public static IEnumerable<CodeInstruction> EmptyMethod(IEnumerable<CodeInstruction> instructions) => 
+		public static IEnumerable<CodeInstruction> EmptyMethod(IEnumerable<CodeInstruction> _) => 
```

Comment:
you sure harmony doesn't need it to be called "instructions" ?

Reply:
Yep